### PR TITLE
test: Use codespell in checkpatch.

### PR DESCRIPTION
**Pull request by ashleypittman**
Signed-off-by: Ashley Pittman <ashley.m.pittman@intel.com>

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -38,7 +38,7 @@
  */
 // To use a test branch (i.e. PR) until it lands to master
 // I.e. for testing library changes
-//@Library(value="pipeline-lib@your_branch") _
+//@Library(value="pipeline-lib@ampittma-codespell") _
```
Comment (brianjmurrell) [inline Jenkinsfile:41]: So this isn't needed any more due to the alternate approach in daos-stack/code_review#21?  Are you sure that having it here is not interfering (as in making the code_review changes work where without it they will fail) with this PR?

Comment (ashleypittman) [reply Jenkinsfile:41]: Correct, this is a "try things and see what works" PR, it's still commented out here, all that's changed is the branch name in the comment so I'm sure it won't have impacted other changes.

Comment (brianjmurrell) [reply Jenkinsfile:41]: Ah yes.  I didn't notice the `//` at the start.  I guess that line should be removed before landing.

Comment (ashleypittman) [reply Jenkinsfile:41]: This PR should be abandoned now, and we can proceed with https://github.com/daos-stack/daos/pull/2643

All that's needed in that one is the code-review one to land, and codespell to be added to the pip install line, which I think it already is.

**Pull request closed without merging**